aws/rust-runtime/aws-http/src/retry.rs

```diff
@@ -39,41 +28,38 @@ const THROTTLING_ERRORS: &[&str] = &[
 ];
 const TRANSIENT_ERRORS: &[&str] = &["RequestTimeout", "RequestTimeoutException"];
 
-impl AwsErrorRetryPolicy {
-    /// Create an `AwsErrorRetryPolicy` with the default set of known error & status codes
+/// Implementation of [`ClassifyResponse`] that models AWS error codes.
+///
+/// In order of priority:
+/// 1. The `x-amz-retry-after` header is checked
+/// 2. The modeled error retry mode is checked
+/// 3. The code is checked against a predetermined list of throttling errors & transient error codes
+/// 4. The status code is checked against a predetermined list of status codes
+#[non_exhaustive]
+#[derive(Clone, Debug)]
+pub struct AwsResponseClassifier;
+
+impl AwsResponseClassifier {
+    /// Create an `AwsResponseClassifier` with the default set of known error & status codes
```

Comment:
If there is a way to link to the private constants with rustdoc, then I could add this pretty easily. I'm reluctant to duplicate the list in the documentation since it will get out of date.